pwndbg/exception.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pdb
```

Comment:
Would be nice to have here a try to import ipdb ( https://pypi.python.org/pypi/ipdb ):
```python
try:
    import ipdb as pdb
except ImportError:
    import pdb
```